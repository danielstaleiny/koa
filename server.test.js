const mongoose = require("./mongoose");
const request = require("supertest");
const app = require("./server");

const User = require("./user");

beforeAll(async () => {
    console.log("index beforeall");
    await User.remove({}).exec();
});
test("root route", async () => {
    const response = await request(app.callback()).get("/users/5");
    expect(response.body).toMatchSnapshot();
});

afterAll(async () => {
    console.log("index afterall");
    mongoose.mongoose.connection.close(function() {
        console.log("Connection with MongoDB closed");
    });
});
