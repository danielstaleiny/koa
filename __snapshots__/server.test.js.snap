// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root route 1`] = `
Object {
  "id": "5",
  "text": "ni8ceee",
}
`;
